src/components/m-button/index.js

```diff
@@ -16,7 +16,7 @@ class AXAButton extends BaseComponentGlobal {
     ghost: PropTypes.bool,
     motion: PropTypes.bool,
     size: PropTypes.oneOf(['sm', 'md', 'lg']),
-    tag: PropTypes.oneOf(['a', 'button']),
+    tag: PropTypes.oneOf(['a', 'button', 'submit']),
```

Comment:
@LucaMele 
Watch out. This is `tag`!
I guess you meant `type=submit|buttom`?

Reply:
https://github.com/axa-ch/patterns-library/pull/796